ItemVendorLocation/ItemVendorLocation.csproj

```diff
@@ -15,9 +15,8 @@
 
 	<ItemGroup>
 		<PackageReference Include="CheapLoc" Version="1.1.8" />
-		<PackageReference Include="Dalamud.DrunkenToad" Version="1.8.15" />
-		<PackageReference Include="DalamudPackager" Version="2.1.12" />
-		<PackageReference Include="XivCommon" Version="9.0.0" />
+		<PackageReference Include="Dalamud.DrunkenToad" Version="1.9.2" />
```

Comment:
I think this can be removed since I don't see anything from this package is used